packedForest/src/forestTypes/unsupervisedForests/urf/urfTree.h

```diff
@@ -0,0 +1,267 @@
+#ifndef urfTree_h
```

Comment:
I think you may have used the rerfTree as a base for this class.  It was a bit confusing at first as to why there were so many changes in the `diff`.  Was there any reason for that?

Reply:
I didn't understand your question.

Reply:
I would have expected `urfTree_h` to be based off of `rfTree`.  But you seem to have based it off `rerfTree`